### PR TITLE
Chore: Adding general and details tab interaction targets

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: src/components/ContentSidebar/SidebarSection.js [hunk 1]
```diff
@@ -75,7 +76,7 @@ class SidebarSection extends PureComponent<Props, State> {
         );
 
         return (
-            <div className={sectionClassName}>
+            <div data-resin-target={interactionTarget} className={sectionClassName}>
```
Comment (priyajeet) [inline src/components/ContentSidebar/SidebarSection.js]: Should be on the button

File: src/components/ContentSidebar/Sidebar.js [hunk 2]
```diff
@@ -152,8 +153,15 @@ const Sidebar = ({
...
         <TabView defaultSelectedIndex={shouldShowSkills ? 0 : 1}>
-            <Tab title={intl.formatMessage(messages.sidebarDetailsTitle)}>{Details}</Tab>
-            <Tab title={intl.formatMessage(messages.activityFeedTitle)}>{ActivityFeedSidebar}</Tab>
+            <Tab
+                title={intl.formatMessage(messages.sidebarDetailsTitle)}
+                data-resin-target={TAB_TARGETS.SELECT_DETAILS}
+            >
+                {Details}
+            </Tab>
+            <Tab title={intl.formatMessage(messages.activityFeedTitle)} data-resin-target={TAB_TARGETS.SELECT_ACTIVITY}>
```
Comment (priyajeet) [inline src/components/ContentSidebar/Sidebar.js]: Does the tab component delegate it?

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #280

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/interactionTargets.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+export const SECTION_TARGETS = {
+    FILE_PROPERTIES: 'details-section-fileproperties',
+    ACCESS_STATS: 'details-section-accessstats'
+};
+
+export const DETAILS_TARGETS = {
+    ACCESS_STATS: {
+        COMMENTS: 'details-accessstats-comments',
```
[the remark this answers is not in the document]
Comment (jeremypress) [reply src/interactionTargets.js]: agreed, I'll double check with Brad but we should still be specific enough if they pass in a component name.

File: src/components/ContentSidebar/Sidebar.js (hunk 2 above)
Comment (jeremypress) [reply src/components/ContentSidebar/Sidebar.js]: yeah it delegates it.

```                    const buttonProps = omit(tab.props, [
                        'className',
                        'children',
                        'title',
                    ]);```

**Pull request merged**